src/main/java/at/ac/tuwien/damap/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -53,244 +53,273 @@ public void determinteDatasetIDs(){
+            contactItems.add(contactName);
+        }
+
+        if (contact.getMbox() != null) {
+            contactMail = contact.getMbox();
+            contactItems.add(contactMail);
+        }
+
+        contactId = getContributorPersonIdentifier(contact);
+        if (contactId != null) {
+            contactItems.add(contactId);
         }
 
-        //mapping project coordinator information
+        if (contact.getAffiliation() != null) {
+            contactAffiliation = contact.getAffiliation();
+            contactItems.add(contactAffiliation);
+        }
+
+        contactAffiliationId = getContributorAffiliationIdentifier(contact);
+        if (contactAffiliationId != null) {
+            contactItems.add(contactAffiliationId);
+        }
+
+        addReplacement(replacements, "[contact]", multipleVariable(contactItems));
+    }
+
+    private void projectCoordinatorInformation() {
+        // mapping project coordinator information
         List<String> coordinatorProperties = new ArrayList<>();
         String coordinatorIdentifierId = "";
         String coordinatorAffiliationIdentifierId = "";
+        
+        if (projectCoordinator == null) {
+            addReplacement(replacements, "[coordinator]", multipleVariable(coordinatorProperties));
+            return;
+        }
 
-        //mapping project coordinator information
-        if (projectCoordinator != null) {
-            if (projectCoordinator.getFirstName() != null && projectCoordinator.getLastName() != null)
-                coordinatorProperties.add(projectCoordinator.getFirstName() + " " + projectCoordinator.getLastName());
+        if (projectCoordinator.getFirstName() != null && projectCoordinator.getLastName() != null)
+            coordinatorProperties.add(projectCoordinator.getFirstName() + " " + projectCoordinator.getLastName());
 
-            if (projectCoordinator.getMbox() != null)
-                coordinatorProperties.add(projectCoordinator.getMbox());
+        if (projectCoordinator.getMbox() != null)
+            coordinatorProperties.add(projectCoordinator.getMbox());
 
-            if (projectCoordinator.getPersonId() != null) {
-                coordinatorIdentifierId = projectCoordinator.getPersonId().getIdentifier();
+        if (projectCoordinator.getPersonId() != null) {
+            coordinatorIdentifierId = projectCoordinator.getPersonId().getIdentifier();
 
-                if (projectCoordinator.getPersonId().getType().toString().equals("orcid")) {
-                    String coordinatorIdentifierType = "ORCID iD: ";
-                    String coordinatorId = coordinatorIdentifierType + coordinatorIdentifierId;
-                    coordinatorProperties.add(coordinatorId);
-                }
+            if (projectCoordinator.getPersonId().getType().toString().equals("orcid")) {
+                String coordinatorIdentifierType = "ORCID iD: ";
+                String coordinatorId = coordinatorIdentifierType + coordinatorIdentifierId;
```

Comment:
Code Smell -> could be refactored like
`String coordinatorId = "ORCID iD: " + coordinatorIdentifierId;`

Reply:
good catch, thanks